docs(website): wip about mode (#829)
cam-inc/viron: packages/website/docs/04-Advanced-Guides/authentication.md

```

## Authentication Types

There are four types of authentication: `email`, `oauth`, `oauthcallback`, and `signout`. Each authentication has a schema of this:

```json
{
  "type": "email" | "oauth" | "oauthcallback" | "signout";
  "provider": string;
  "operatioId": string; // Used to determine how to send a request.

>   "mode"?: 'navigate' | 'cors'; // Use to determine how to open the Oauth endpoint. Only if type is oauth.

Reviewer: ```suggestion
  "mode"?: 'navigate' | 'cors'; // Used to determine the method of opening the Oauth endpoint. Only applicable if type is 'oauth'.

```

  "defaultParametersValue"?: any;
  "defaultRequestBodyValue"?: any;
}
```

:::tip　
Basically, you should set `mode: 'cors'`.

> However, if you are self-hosting viron, you can choose any mode.

Reviewer: Typically, you should set `mode: 'cors'`.
However, if you are self-hosting Viron, you have the flexibility to choose any mode.

> Basically, you should set `mode: 'cors'`.
> However, if you are self-hosting viron, you can choose any mode.

Author: ```suggestion
Typically, you should set mode: 'cors'.
However, if you are self-hosting Viron, you have the flexibility to choose any mode.
```
